core/src/main/scala/kafka/server/KafkaApis.scala

```diff
@@ -1333,6 +1335,40 @@ class KafkaApis(val requestChannel: RequestChannel,
     }
   }
 
+  def handleCreatePartitionsRequest(request: RequestChannel.Request): Unit = {
+    val alterPartitionCountsRequest = request.body[CreatePartitionsRequest]
+    var valid: Map[String, NewPartitions] = null
+    var errors: Map[String, ApiError] = null
```

Comment:
There's no real reason for declaring these vars. `if/else` in Scala are expressions, so they can be assigned as vals. But I wonder why not follow the approach in `createTopics`. All things being equal, it's better to be consistent. If there are good reasons to deviate, then it's fine to do so.